from PyQt5 import QtCore, QtGui, QtWidgets

class Ui_InferenciaTipos(object):
    def setupUi(self, InferenciaTipos):
        InferenciaTipos.setObjectName("InferenciaTipos")
        InferenciaTipos.setWindowModality(QtCore.Qt.WindowModal)
        InferenciaTipos.resize(1350, 900)
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap("images/Qt.ico"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        InferenciaTipos.setWindowIcon(icon)
        InferenciaTipos.setTabShape(QtWidgets.QTabWidget.Rounded)
        self.centralwidget = QtWidgets.QWidget(InferenciaTipos)
        self.centralwidget.setObjectName("centralwidget")
        self.gridLayout_3 = QtWidgets.QGridLayout(self.centralwidget)
        self.gridLayout_3.setObjectName("gridLayout_3")
        self.tabWidgetCodigo = QtWidgets.QTabWidget(self.centralwidget)
        font = QtGui.QFont()
        font.setFamily("Tahoma")
        font.setPointSize(12)
        font.setBold(False)
        font.setWeight(50)
        self.tabWidgetCodigo.setFont(font)
        self.tabWidgetCodigo.setObjectName("tabWidgetCodigo")
        self.tabCodigoCOOL = QtWidgets.QWidget()
        self.tabCodigoCOOL.setObjectName("tabCodigoCOOL")
        self.gridLayout = QtWidgets.QGridLayout(self.tabCodigoCOOL)
        self.gridLayout.setObjectName("gridLayout")
        self.textCodigoCOOL = QtWidgets.QPlainTextEdit(self.tabCodigoCOOL)
        font = QtGui.QFont()
        font.setPointSize(18)
        self.textCodigoCOOL.setFont(font)
        self.textCodigoCOOL.setObjectName("textCodigoCOOL")
        self.gridLayout.addWidget(self.textCodigoCOOL, 0, 0, 1, 1)
        self.tabWidgetCodigo.addTab(self.tabCodigoCOOL, "")
        self.tabResultados = QtWidgets.QWidget()
        self.tabResultados.setObjectName("tabResultados")
        self.gridLayout_2 = QtWidgets.QGridLayout(self.tabResultados)
        self.gridLayout_2.setObjectName("gridLayout_2")
        self.textResultados = QtWidgets.QPlainTextEdit(self.tabResultados)
        font = QtGui.QFont()
        font.setPointSize(18)
        self.textResultados.setFont(font)
        self.textResultados.setUndoRedoEnabled(False)
        self.textResultados.setLineWrapMode(QtWidgets.QPlainTextEdit.NoWrap)
        self.textResultados.setReadOnly(True)
        self.textResultados.setObjectName("textResultados")
        self.gridLayout_2.addWidget(self.textResultados, 0, 0, 1, 1)
        self.tabWidgetCodigo.addTab(self.tabResultados, "")
        self.gridLayout_3.addWidget(self.tabWidgetCodigo, 0, 0, 1, 1)
        InferenciaTipos.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(InferenciaTipos)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 1350, 22))
        self.menubar.setObjectName("menubar")
        self.menuInicio = QtWidgets.QMenu(self.menubar)
        self.menuInicio.setObjectName("menuInicio")
        self.menuAnalisis = QtWidgets.QMenu(self.menubar)
        self.menuAnalisis.setObjectName("menuAnalisis")
        self.menuOpciones = QtWidgets.QMenu(self.menubar)
        self.menuOpciones.setObjectName("menuOpciones")
        InferenciaTipos.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(InferenciaTipos)
        font = QtGui.QFont()
        font.setFamily("Tahoma")
        font.setPointSize(12)
        self.statusbar.setFont(font)
        self.statusbar.setObjectName("statusbar")
        InferenciaTipos.setStatusBar(self.statusbar)
        self.actionNuevoCodigo = QtWidgets.QAction(InferenciaTipos)
        icon1 = QtGui.QIcon()
        icon1.addPixmap(QtGui.QPixmap("images/selection-input.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionNuevoCodigo.setIcon(icon1)
        self.actionNuevoCodigo.setObjectName("actionNuevoCodigo")
        self.actionGuardarResultados = QtWidgets.QAction(InferenciaTipos)
        icon2 = QtGui.QIcon()
        icon2.addPixmap(QtGui.QPixmap("images/disk--pencil.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionGuardarResultados.setIcon(icon2)
        self.actionGuardarResultados.setObjectName("actionGuardarResultados")
        self.actionCargarCodigo = QtWidgets.QAction(InferenciaTipos)
        icon3 = QtGui.QIcon()
        icon3.addPixmap(QtGui.QPixmap("images/blue-folder-horizontal-open.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionCargarCodigo.setIcon(icon3)
        self.actionCargarCodigo.setObjectName("actionCargarCodigo")
        self.actionSalir = QtWidgets.QAction(InferenciaTipos)
        icon4 = QtGui.QIcon()
        icon4.addPixmap(QtGui.QPixmap("images/cross-circle.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionSalir.setIcon(icon4)
        self.actionSalir.setObjectName("actionSalir")
        self.actionHelp = QtWidgets.QAction(InferenciaTipos)
        icon5 = QtGui.QIcon()
        icon5.addPixmap(QtGui.QPixmap("images/question.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionHelp.setIcon(icon5)
        self.actionHelp.setObjectName("actionHelp")
        self.actionAyuda = QtWidgets.QAction(InferenciaTipos)
        icon6 = QtGui.QIcon()
        icon6.addPixmap(QtGui.QPixmap("images/question.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionAyuda.setIcon(icon6)
        self.actionAyuda.setObjectName("actionAyuda")
        self.actionAcercaInferenciaTipos = QtWidgets.QAction(InferenciaTipos)
        icon7 = QtGui.QIcon()
        icon7.addPixmap(QtGui.QPixmap("images/lifebuoy.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionAcercaInferenciaTipos.setIcon(icon7)
        self.actionAcercaInferenciaTipos.setObjectName("actionAcercaInferenciaTipos")
        self.actionSaveGrammarAs = QtWidgets.QAction(InferenciaTipos)
        icon8 = QtGui.QIcon()
        icon8.addPixmap(QtGui.QPixmap("images/disk--pencil.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionSaveGrammarAs.setIcon(icon8)
        self.actionSaveGrammarAs.setObjectName("actionSaveGrammarAs")
        self.actionAnalyse = QtWidgets.QAction(InferenciaTipos)
        icon9 = QtGui.QIcon()
        icon9.addPixmap(QtGui.QPixmap("images/control.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionAnalyse.setIcon(icon9)
        font = QtGui.QFont()
        font.setPointSize(8)
        self.actionAnalyse.setFont(font)
        self.actionAnalyse.setObjectName("actionAnalyse")
        self.actionQuickAnalyse = QtWidgets.QAction(InferenciaTipos)
        icon10 = QtGui.QIcon()
        icon10.addPixmap(QtGui.QPixmap("images/controlx2.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionQuickAnalyse.setIcon(icon10)
        self.actionQuickAnalyse.setObjectName("actionQuickAnalyse")
        self.actionAnalisis = QtWidgets.QAction(InferenciaTipos)
        self.actionAnalisis.setIcon(icon9)
        self.actionAnalisis.setObjectName("actionAnalisis")
        self.actionQuickAnalyse_2 = QtWidgets.QAction(InferenciaTipos)
        icon11 = QtGui.QIcon()
        icon11.addPixmap(QtGui.QPixmap("images/controlx2.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionQuickAnalyse_2.setIcon(icon11)
        self.actionQuickAnalyse_2.setObjectName("actionQuickAnalyse_2")
        self.actionAcercaAutores = QtWidgets.QAction(InferenciaTipos)
        icon12 = QtGui.QIcon()
        icon12.addPixmap(QtGui.QPixmap("images/lifebuoy.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionAcercaAutores.setIcon(icon12)
        self.actionAcercaAutores.setObjectName("actionAcercaAutores")
        self.actionGuardarCodigo = QtWidgets.QAction(InferenciaTipos)
        icon13 = QtGui.QIcon()
        icon13.addPixmap(QtGui.QPixmap("images/disk.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionGuardarCodigo.setIcon(icon13)
        self.actionGuardarCodigo.setObjectName("actionGuardarCodigo")
        self.actionOrientacion = QtWidgets.QAction(InferenciaTipos)
        icon14 = QtGui.QIcon()
        icon14.addPixmap(QtGui.QPixmap("images/clipboard-paste-document-text.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionOrientacion.setIcon(icon14)
        self.actionOrientacion.setObjectName("actionOrientacion")
        self.actionInforme = QtWidgets.QAction(InferenciaTipos)
        icon15 = QtGui.QIcon()
        icon15.addPixmap(QtGui.QPixmap("images/printer.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.actionInforme.setIcon(icon15)
        self.actionInforme.setObjectName("actionInforme")
        self.menuInicio.addSeparator()
        self.menuInicio.addAction(self.actionNuevoCodigo)
        self.menuInicio.addSeparator()
        self.menuInicio.addAction(self.actionCargarCodigo)
        self.menuInicio.addSeparator()
        self.menuInicio.addAction(self.actionGuardarCodigo)
        self.menuInicio.addAction(self.actionGuardarResultados)
        self.menuInicio.addSeparator()
        self.menuInicio.addAction(self.actionSalir)
        self.menuAnalisis.addAction(self.actionAnalisis)
        self.menuOpciones.addAction(self.actionAyuda)
        self.menuOpciones.addSeparator()
        self.menuOpciones.addAction(self.actionOrientacion)
        self.menuOpciones.addAction(self.actionInforme)
        self.menuOpciones.addSeparator()
        self.menuOpciones.addAction(self.actionAcercaAutores)
        self.menubar.addAction(self.menuInicio.menuAction())
        self.menubar.addAction(self.menuAnalisis.menuAction())
        self.menubar.addAction(self.menuOpciones.menuAction())

        self.retranslateUi(InferenciaTipos)
        self.tabWidgetCodigo.setCurrentIndex(0)
        QtCore.QMetaObject.connectSlotsByName(InferenciaTipos)

    def retranslateUi(self, InferenciaTipos):
        _translate = QtCore.QCoreApplication.translate
        InferenciaTipos.setWindowTitle(_translate("InferenciaTipos", "Inferencia de Tipos"))
        self.tabWidgetCodigo.setTabText(self.tabWidgetCodigo.indexOf(self.tabCodigoCOOL), _translate("InferenciaTipos", "CÃ³digo de COOL"))
        self.tabWidgetCodigo.setTabText(self.tabWidgetCodigo.indexOf(self.tabResultados), _translate("InferenciaTipos", "Resultados"))
        self.menuInicio.setTitle(_translate("InferenciaTipos", "Inicio"))
        self.menuAnalisis.setTitle(_translate("InferenciaTipos", "AnÃ¡lisis"))
        self.menuOpciones.setTitle(_translate("InferenciaTipos", "Opciones"))
        self.actionNuevoCodigo.setText(_translate("InferenciaTipos", "Nuevo cÃ³digo"))
        self.actionNuevoCodigo.setStatusTip(_translate("InferenciaTipos", "Nuevo cÃ³digo"))
        self.actionNuevoCodigo.setShortcut(_translate("InferenciaTipos", "Ctrl+N"))
        self.actionGuardarResultados.setText(_translate("InferenciaTipos", "Guardar resultados"))
        self.actionGuardarResultados.setStatusTip(_translate("InferenciaTipos", "Guardar resultados"))
        self.actionGuardarResultados.setShortcut(_translate("InferenciaTipos", "Ctrl+R"))
        self.actionCargarCodigo.setText(_translate("InferenciaTipos", "Cargar cÃ³digo"))
        self.actionCargarCodigo.setStatusTip(_translate("InferenciaTipos", "Cargar cÃ³digo"))
        self.actionCargarCodigo.setShortcut(_translate("InferenciaTipos", "Ctrl+O"))
        self.actionSalir.setText(_translate("InferenciaTipos", "Salir"))
        self.actionSalir.setStatusTip(_translate("InferenciaTipos", "Salir"))
        self.actionSalir.setShortcut(_translate("InferenciaTipos", "Ctrl+F4"))
        self.actionHelp.setText(_translate("InferenciaTipos", "Ayuda"))
        self.actionHelp.setStatusTip(_translate("InferenciaTipos", "Â¿CÃ³mo usar?"))
        self.actionHelp.setShortcut(_translate("InferenciaTipos", "F1"))
        self.actionAyuda.setText(_translate("InferenciaTipos", "Ayuda"))
        self.actionAyuda.setStatusTip(_translate("InferenciaTipos", "Ayuda"))
        self.actionAyuda.setShortcut(_translate("InferenciaTipos", "F1"))
        self.actionAcercaInferenciaTipos.setText(_translate("InferenciaTipos", "Acerca de Inferencia de Tipos"))
        self.actionAcercaInferenciaTipos.setStatusTip(_translate("InferenciaTipos", "Acerca de Inferencia de Tipos"))
        self.actionSaveGrammarAs.setText(_translate("InferenciaTipos", "Guardar gramÃ¡tica como ..."))
        self.actionSaveGrammarAs.setStatusTip(_translate("InferenciaTipos", "Guardar GramÃ¡tica Como"))
        self.actionAnalyse.setText(_translate("InferenciaTipos", "Analizar"))
        self.actionAnalyse.setToolTip(_translate("InferenciaTipos", "Analizar"))
        self.actionAnalyse.setStatusTip(_translate("InferenciaTipos", "Analizar"))
        self.actionAnalyse.setShortcut(_translate("InferenciaTipos", "F5"))
        self.actionQuickAnalyse.setText(_translate("InferenciaTipos", "AnÃ¡lisis rÃ¡pido"))
        self.actionQuickAnalyse.setToolTip(_translate("InferenciaTipos", "AnÃ¡lisis rÃ¡pido"))
        self.actionQuickAnalyse.setStatusTip(_translate("InferenciaTipos", "AnÃ¡lisis rÃ¡pido"))
        self.actionQuickAnalyse.setShortcut(_translate("InferenciaTipos", "F9"))
        self.actionAnalisis.setText(_translate("InferenciaTipos", "AnÃ¡lisis"))
        self.actionAnalisis.setStatusTip(_translate("InferenciaTipos", "AnÃ¡lisis"))
        self.actionAnalisis.setShortcut(_translate("InferenciaTipos", "F5"))
        self.actionQuickAnalyse_2.setText(_translate("InferenciaTipos", "AnÃ¡lisis rÃ¡pido de la gramÃ¡tica"))
        self.actionQuickAnalyse_2.setShortcut(_translate("InferenciaTipos", "F9"))
        self.actionAcercaAutores.setText(_translate("InferenciaTipos", "Acerca de los autores"))
        self.actionAcercaAutores.setStatusTip(_translate("InferenciaTipos", "Acerca de los autores"))
        self.actionGuardarCodigo.setText(_translate("InferenciaTipos", "Guardar cÃ³digo"))
        self.actionGuardarCodigo.setStatusTip(_translate("InferenciaTipos", "Guardar cÃ³digo"))
        self.actionGuardarCodigo.setShortcut(_translate("InferenciaTipos", "Ctrl+S"))
        self.actionOrientacion.setText(_translate("InferenciaTipos", "OrientaciÃ³n"))
        self.actionInforme.setText(_translate("InferenciaTipos", "Informe"))
